go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/sigstore/sigstore v1.8.1
 	github.com/sigstore/timestamp-authority v1.2.1
 	github.com/stretchr/testify v1.8.4
-	github.com/theupdateframework/go-tuf/v2 v2.0.0-20240207172116-f5cf71290141
+	github.com/theupdateframework/go-tuf/v2 v2.0.0-20240222081530-454b12158917
```

Comment:
@kommendorkapten not a blocker, but do you know when go-tuf maintainers will be cutting a release with the v2 implementation?

Reply:
@haydentherapper there is still one open bug for Windows in go-tuf that I intend to fix today. When all test cases are working for all platforms, and we have a clear understanding of the current open issues, I think it's time we create a first release. I would guess one-two weeks from now?